### ISSUE TITLE
Aggregated Git highlighting issues (-git.ch chroma problems)

### ISSUE THREAD

**Issue by psprint** [zdharma/fast-syntax-highlighting#89]
* From #71 
[ ] `git -C ~/github/zsh-unique-id.git diff --cached` – doesn't recognize that -C has an argument, fails to highlight correctly
[ ] `git tag -a 'v1.27' -m "Test tag" 24034b1` – the commit SHA isn't highlighted
[ ] `git fetch origin feature/myfeature1; git checkout feature/myfeature1` – the `checkout` will have the branch argument highlighted incorrectly

**Comment by psprint** [comment]
Yes I think it would be posible. By the aliases you mean e.g. "lp = log -p" which should be recognized as `log` subcommand by F-Sy-H, etc.?

**Comment by jcrd** [comment]
Yes, exactly.

**Comment by psprint** [comment]
@jcrd: I've commited the support for the git-aliases. It's simple, but should work in 80% of cases (the problem: aliases that are using options, the ones that a git-chroma analyzes to decide on the highlighting).
